docs/background/energy_offset_array.rst

```diff
@@ -0,0 +1,24 @@
+.. _bg_models:
+
+EnergyOffset Array
+=================
+
+**energyOffset Array** is a 2D array *(energy,offset)* that is filled with an eventlist. For a set of observations, by giving an energy binning and an offset binning, you fill the events in this histogram.
```

Comment:
Replace this with:
```
The `~gammapy.background.EnergyOffsetArray` class represents a 2D array ...
```

It's very convenient to learn if the high-level, narrative docs pages (the rst files in the `docs` folder) and the API docs (generated from the docstrings in the `.py` files are cross-linked.

You could also link from the `EnergyOffsetArray` class docstring to this page by using label and ref.
There's a description here:
http://www.sphinx-doc.org/en/stable/markup/inline.html#cross-referencing-arbitrary-locations

@JouvinLea - If it's not clear how to generate that link, let me know and I'll describe it.